scout/server/blueprints/omics_variants/templates/omics_variants/outliers.html

```diff
@@ -0,0 +1,265 @@
+
+{% block title %}
+  {{ super() }} - {{ institute.display_name }} - {{ case.display_name }} - {{ form.variant_type.data|capitalize }} omics outlier variants
+{% endblock %}
+
+{% block css %}
+  {{ super() }}
+  {{ external_stylesheets() }}
+{% endblock %}
+
+{% block top_nav %}
+  {{ super() }}
+  <li class="nav-item">
+    <a class="nav-link text-nowrap" href="{{ url_for('cases.index') }}">Institutes</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link text-nowrap" href="{{ url_for('overview.cases', institute_id=institute._id) }}">
+      {{ institute.display_name }}
+    </a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link text-nowrap" href="{{ url_for('cases.case', institute_id=institute._id, case_name=case.display_name) }}">
+      {{ case.display_name }}
+    </a>
+  </li>
+  <li class="nav-item active d-flex align-items-center">
+    <span class="navbar-text">{{ form.variant_type.data|capitalize }} WTS outliers</span>
+  </li>
+{% endblock %}
+
+{% block content_main %}
+  <div class="container-float">
+    <form method="POST" id="filters_form" action="{{url_for('omics_variants.outliers', institute_id=institute._id, case_name=case.display_name)}}"
+        enctype="multipart/form-data" onsubmit="return validateForm()">
+
+      {{ pagination_hidden_div(page) }}
+      <div class="card panel-default" id="accordion">
+        <div class="card-header">
+          <strong><a data-bs-toggle="collapse" data-parent="#accordion" href="#collapseFilters">Filters</a></strong>
+        </div>
+        <!--Expand filters form if filters were used in a previous POST request-->
+        <div class="card-body panel-collapse collapse {{ 'show' if expand_search is sameas true }}" id="collapseFilters">
+            {{ outlier_filters(form, institute, case) }}
+        </div>
+        {{ omics_filter_form_footer(form, result_size, total_variants, more_variants, page, variants|length, institute) }}
+      </div>
+    </form>
+    <div class="card mt-3">
+      <table class="table table-hover table-bordered" style="table-layout: fixed">
+        <thead class="thead table-light">
+              <tr>
+                <th title="Variant display name">Outlier</th>
+                <th style="width:7%" title="Sub-category">Type</th>
+                <th style="width:14%" title="HGNC symbols">Gene</th>
+                <th title="Functional annotation">Func. annotation</th>
+                <th style="width:7%" title="Value - delta Psi or l2fc ">Value</th>
+                <th style="width:7%" title="P-value">P-value</th>
+                <th style="width:10%" title="Individual">Ind</th>
+                <th style="width:3%" title="Chromosome">Chr</th>
+                <th title="Position">Pos</th>
+              </tr>
+        </thead>
+          <tbody>
+            {% for variant in variants %}
+              <tr>
+              <td>{{ variant.display_name|truncate(36,True) }}</td>
```

Comment:
Replace underscores with space perhaps? Looks like this otherwise:

<img width="247" alt="image" src="https://github.com/user-attachments/assets/1a240912-ae3f-4ac1-9239-944f65e0415d">

Reply:
Yes, question is if we should even show it at all in this form. Since we dont go to variant pages or rank them right now, I currently feel this could be hidden.

Reply:
Might be informative no?

Reply:
I think all the constituent substrings would be elsewhere on the line as well. I'll check before removing.